tests/integrations/android_sdk/test_verify_android_sdk.py

```diff
@@ -0,0 +1,281 @@
+import os
+import shutil
+import sys
+from unittest.mock import MagicMock
+
+import pytest
+from requests import exceptions as requests_exceptions
+
+from briefcase.exceptions import BriefcaseCommandError, NetworkFailure
+from briefcase.integrations.android_sdk import verify_android_sdk
+
+
+@pytest.fixture
+def mock_command(tmp_path):
+    command = MagicMock()
+
+    # Mock-out the `sys` module so we can mock out the Python version in some tests.
+    command.sys = MagicMock()
+
+    # Use the `tmp_path` in `dot_briefcase_path` to ensure tests don't interfere
+    # with each other.
+    command.dot_briefcase_path = tmp_path / ".briefcase"
+
+    # Make the `os` module and `host_os` live.
+    command.os = os
```

Comment:
I'd prefer not to provide the entire `os` module unless you need the whole module, but it's OK.

Reply:
What's your reasoning here? This is essentially only restoring "default behavior"; i.e., that os exists and is usable. Generally, my preference is that is mocks *can* be avoided, they should be. Admittedly they're unavoidable in a lot of circumstances, but when they *can* be avoided, we might as well - that's one less discrepancy between production and test.